apiserver/common/firewall/firewall.go

```diff
@@ -51,7 +51,7 @@ func WatchEgressAddressesForRelations(resources facade.Resources, st State, mode
 		//filter := func(id interface{}) bool {
 		//	include, err := includeAsIngressSubnet(id.(string))
 		//	if err != nil {
-		//		logger.Warningf(context.TODO(), "invalid CIDR %q", id)
+		//		logger.Warningf(ctx, "invalid CIDR %q", id)
```

Comment:
😱

Reply:
sed script working just as expected :rofl:

Reply:
It's just a sed script.